pkg/controller/atlas/provider.go

```diff
@@ -76,7 +76,8 @@ func (p *ProductionProvider) IsResourceSupported(resource api.AtlasCustomResourc
 		*akov2.AtlasDatabaseUser,
 		*akov2.AtlasSearchIndexConfig,
 		*akov2.AtlasBackupCompliancePolicy,
-		*akov2.AtlasFederatedAuth:
+		*akov2.AtlasFederatedAuth,
+		*akov2.AtlasPrivateEndpoint:
```

Comment:
@igor-karpukhin we probably need to add custom roles in your case and network peering in mine here.
And also make sure to include some testing about gov support.